ocfweb/component/markdown.py

```diff
@@ -14,24 +20,37 @@
 # tags of a format like: [[!meta title="Backups"]]
 META_REGEX = re.compile(r'\[\[!meta ([a-z]+)="([^"]*)"\]\]')
 
+# Make mypy play nicely with mixins https://github.com/python/mypy/issues/5837
```

Comment:
A fix for this was actually just merged (https://github.com/python/mypy/pull/7860) so if you could leave a todo, to remove the work around once we update mypy that'd be great!

Reply:
@64bitpandas @ethanhs can this be removed now?

Reply:
Yup! Thank you for following up on this, mypy 0.750 just came out so it should be possible to remove this.

Reply:
I tried to upgrade to .750 but this seems to break mypy_django_plugin:

```
Traceback (most recent call last):
  File "venv/bin/mypy", line 11, in <module>
    sys.exit(console_entry())
  File "/home/b/be/bencuan/ocfweb/venv/lib/python3.7/site-packages/mypy/__main__.py", line 8, in console_entry
    main(None, sys.stdout, sys.stderr)
  File "mypy/main.py", line 89, in main
  File "mypy/build.py", line 166, in build
  File "mypy/build.py", line 235, in _build
  File "mypy/build.py", line 2567, in dispatch
  File "mypy/build.py", line 2725, in load_graph
  File "mypy/build.py", line 1826, in __init__
  File "mypy/build.py", line 2101, in compute_dependencies
  File "mypy/plugin.py", line 785, in get_additional_deps
  File "mypy/plugin.py", line 711, in get_additional_deps
  File "/home/b/be/bencuan/ocfweb/venv/lib/python3.7/site-packages/mypy_django_plugin/main.py", line 124, in get_additional_deps
    if file.fullname() == 'django.conf' and self.django_context.django_settings_module:
TypeError: 'str' object is not callable
Makefile:26: recipe for target 'mypy' failed
make: *** [mypy] Error 1
```
I tried upgrading django and django-stubs to latest and the error didn't change. I'm not sure which codebase the error originates from?

Reply:
Ah, yeah that is an issue. I'll see if I can make a PR to fix the plugin with the changes made in this PR to mypy https://github.com/python/mypy/pull/7829